engine/redis_v2/timer.go

```diff
@@ -79,12 +88,24 @@ func (t *Timer) Name() string {
 	return t.name
 }
 
+func encodeScore(timestamp int64, tries uint16) float64 {
+	return float64(((timestamp & 0xffffffff) << 16) | int64(tries&0xffff))
+}
+
+func decodeScore(score float64) (int64, uint16) {
+	val := int64(score)
+	timestamp := (val >> 16) & 0xffffffff
+	tries := uint16(val & 0xffff)
+	return timestamp, tries
+}
+
 func (t *Timer) Add(namespace, queue, jobID string, delaySecond uint32, tries uint16) error {
 	metrics.timerAddJobs.WithLabelValues(t.redis.Name).Inc()
 	timestamp := time.Now().Unix() + int64(delaySecond)
 
-	// struct-pack the data in the format `Hc0Hc0HHc0`:
-	//   {namespace len}{namespace}{queue len}{queue}{tries}{jobID len}{jobID}
+	score := encodeScore(timestamp, tries)
```

Comment:
1. I would suggest adding an output format example for this function.
2. The comments below seem to be irrelevant.